tests/test_cli_omx_hud_tmux_split.py

```
@@ -0,0 +1,340 @@
+
+    proc = run_cli(cli_path, socket_path, fake_home, omx_hud_split_args(cwd))
+    if proc.returncode != 0:
+        raise AssertionError(
+            "disabled HUD split returned non-zero\n"
+            f"stdout={proc.stdout.strip()}\n"
+            f"stderr={proc.stderr.strip()}"
+        )
+    if proc.stdout.strip():
+        raise AssertionError(f"disabled HUD split should not print a pane id: {proc.stdout!r}")
+    if state.split_params:
+        raise AssertionError(f"disabled HUD should not create a split: {state.split_params!r}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)